api/app/users/tests.py

```diff
@@ -594,9 +594,7 @@ def test_no_response_and_invited(self):
 
         # Assert that request succeeds and no users are returned.
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.json, [])
-
-        self.assertTrue(False)
+        self.assertEqual(len(response.json), 1)
```

Comment:
Ideally we should check the contents of the response as well. At least check the uid of the user that is returned to make sure it's the right one.

Reply:
Okay, great. So, as I have it here, it's just checking that there is a response, and the length thereof?
When you say the contents, do you mean type? I hope that's the right word - but, do you mean check if it's a list / dictionary?